File: grade.java
import java.util.*;

class grade
{
	static char alpha(int marks)
	{
		if(marks>=90)
			return 'S';
		else if(marks>=80)
			return 'A';
		else if(marks>=70)
			return 'B';
		else if(marks>=60)
			return 'C';
		else if(marks>=50)
			return 'D';
		else
			return 'F';
	}




	public static void main(String[] args)
	{
		Scanner sn = new Scanner(System.in);
		System.out.print("Enter the number of subjects:	");
		int n = sn.nextInt();
		int i = 0;
		int cie, see;
		int total[] = new int[n];
		while(i<n)
		{
			System.out.print("Enter the C.I.E. marks in Subject "+(i+1)+":  ");
			cie = sn.nextInt();
			System.out.print("\nEnter the S.E.E. marks in Subject "+(i+1)+":  ");
			see = sn.nextInt();
			total[i] = cie + see/2;
			i++;
		}
		i = 0;
		while(i<n)
		{
			System.out.println("Grade in Subject "+(i+1)+" is "+alpha(total[i]));
			i++;
		}
	}
}

File: C files/grade.c
#include<stdio.h>

int main()
{
	float cie, see, sum;
	printf("Enter cie marks:	");
	scanf("%f", &cie);
	printf("\nEnter SEE marks:	");
	scanf("%f", &see);
	see = see/2;
	sum = cie + see;
	if (sum>= 90)
		printf("\n\t Your Grade is: S");
	else if(sum>=80)
		printf("\n\tYour Grade is: A");
	else if(sum>=70)
		printf("Your Grade is: B");
	else if(sum>=60)
		printf("Your Grade is: C");
	else if(sum>=50)
		printf("Your Grade is: D");
	else
		printf("Your Grade is: F");
	printf("\n");
}

